mk

```diff
@@ -54,7 +54,7 @@ case $1 in
             "reload")
                 shift 2
                 ./mk docker rmall;
-                ./mk docker up $@;
+                ./mk docker up --force-recreate --build $@;
```

Comment:
update help function according to changes

Reply:
Done in [this commit](https://github.com/samuel-joly/Montcelard-event/commit/57719fa53c636d312c2469b2b461a61c12922e09)